Core.ApplicationServices/Helpers/CronPatternDefaults.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Core.ApplicationServices.Helpers
+{
+    public static class CronPatternDefaults
+    {
+        public const int TriggerHourUTC = 8;
```

Comment:
Hvad med gamle advis i DB som er oprettet før det blev sat til at være 08 de skulle sendes? De sendes vel stadig på det givne tidpunkt. (Mener det er 00)

Reply:
Ja med mindre man retter den for så rescheduleres de og der vil den så se på startdatoen så vi bør ikke få nogle problemer. Det er primært første-afsendelse der vil være udfordringen